dev-libs/gtest/patches/gtest-1.8.0.patchset

```diff
@@ -0,0 +1,82 @@
+From 08c0526901283c24310e60fd8fdac8023c41e293 Mon Sep 17 00:00:00 2001
+From: begasus <begasus@gmail.com>
+Date: Sat, 13 Jan 2018 12:14:47 +0100
+Subject: Fix use for GnuInstallDirs
+
+
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 8d2b552..7a86b74 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -4,10 +4,22 @@ project( googletest-distribution )
+ 
+ enable_testing()
+ 
++include(CMakeDependentOption)
++if (CMAKE_VERSION VERSION_LESS 2.8.5)
++  set(CMAKE_INSTALL_BINDIR "bin" CACHE STRING "User executables (bin)")
++  set(CMAKE_INSTALL_LIBDIR "lib${LIB_SUFFIX}" CACHE STRING "Object code libraries (lib)")
```

Comment:
This might work, but you are hardcoding, and this is not recommended.

BTW, I think @mmlr will confirm that `$relativeLibDir` should not be hardcoded in a patch but, instead, passed from the recipe.

Reply:
Using -DCMAKE_INSTALL_LIBDIR=$libDir" would be better to use in the recipe then?

Reply:
Yep, I think so. BTW, `dev-qt/prison/` does so:
```
		-DCMAKE_INSTALL_PREFIX:PATH="$prefix" \
		-DCMAKE_INSTALL_LIBDIR:PATH="$libDir" \
		-DCMAKE_INSTALL_INCLUDEDIR:PATH="$includeDir"
```

Reply:
Yeah, they probably work because they are mentioned in the cmake file(s), without the GnuInstallDirs it won't work, need to investigate ... :)

Reply:
One question (well maybe) :) should I keep the patch and use -DCMAKE_INSTALL_* for the others in the recipe?

Reply:
IDK. I'd have to look closer...

Reply:
@Begasus 
Don't forget about [`$cmakeDirArgs`][0]

[0]: https://github.com/haikuports/haikuports/issues/2074

Reply:
@alaviss is this needed? we are way past version 2.8.5 already?
Taking a quick look I can't find any recipe forcing a version for cmake (could be that I missed) :)